rules/0132.go

```diff
@@ -195,7 +195,12 @@ func checkListResponseMessageName() lint.Rule {
 	}
 }
 
-// Return true if this is a List method, false otherwise.
+// Return true if this is an AIP-132 List method, false otherwise.
 func isListMethod(m p.MethodDescriptor) bool {
```

Comment:
There is a lot of duplication throughout all this code. I wonder if some utility methods might help. E.g. https://play.golang.org/p/qFgJXFdDrMV